### ISSUE TITLE
[Bug] [Overlays] `NullReferenceException` While Selecting Stash Tab

### ISSUE THREAD

**Issue by gulenceset** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#560]
Reporting an exception I received while using Chaos Recipe Enhancer v3.23.1005:
```
The following exception occurred: Dispatcher.UnhandledExceptionSystem.NullReferenceException: Object reference not set to an instance of an object.
   at ChaosRecipeEnhancer.UI.Windows.StashTabOverlayWindow.GenerateReconstructedStashTabsFromApiResponse() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Windows\StashTabOverlayWindow.xaml.cs:line 259
   at ChaosRecipeEnhancer.UI.Windows.StashTabOverlayWindow.Show() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Windows\StashTabOverlayWindow.xaml.cs:line 84
   at ChaosRecipeEnhancer.UI.Windows.SetTrackerOverlayWindow.RunStashTabOverlay() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Windows\SetTrackerOverlayWindow.xaml.cs:line 150
   at ChaosRecipeEnhancer.UI.UserControls.SetTrackerOverlayDisplays.StandardDisplay.OnStashTabOverlayButtonClicked(Object sender, RoutedEventArgs e) in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\UserControls\SetTrackerOverlayDisplays\StandardDisplay.xaml.cs:line 18
```

**Comment by gulenceset** [comment]
When i try to select my stash tab getting this error message

**Comment by HiMarioLopez** [comment]
To help out fully can you provide me with some more info? Screenshots of your CRE settings, your full CRE settings file if possible, screenshots of the error happening. Video recordings.. anything

**Comment by HiMarioLopez** [comment]
resolved #582
